TiDB monitoring: add table storage info status api (#6110)
pingcap/docs: tidb-monitoring-api.md

You can use the following two types of interfaces to monitor the TiDB cluster state:

- [The state interface](#running-status): this interface uses the HTTP interface to get the component information.

> - [Storage information](#storage-information): it uses the HTTP interface to get the storage information from data tables.

Reviewer: ```suggestion
- [Storage information](#storage-information): this interface uses the HTTP interface to get the storage information of data tables.
```

}
```

#### Storage information

> The following example uses `http://${host}:${port}/schema_storage/${db}/${table}` to get the storage information from the specific data tables. The result is returned in **JSON** format.

Reviewer: ```suggestion
The following example uses `http://${host}:${port}/schema_storage/${db}/${table}` to get the storage information of the specific data table. The result is returned in **JSON** format.
```
